rfcs/text/0000-conceptual-data-model.md

```diff
@@ -0,0 +1,111 @@
+
+
+The head graph is the graph of the latest version of all the metadata, excluding those that are obsolete due to a restructuring update or those that are flagged as erroneous.  This is the view normally used in selectinzΩg data sets and exploring the atlas.  Traversing from the roots to result in a branching set of paths to the leaves.  Normally, this graph would be generated by traversing via UUID, ignored the version timestamps.
+
+The providence graph gives the exact history of data and metadata as it existed at the time a given process was run.  This view provides the ability to analyze and understand an experiment.  It is a reverse traversal from leaves to roots following FQIDs of the input to each metadata instance as it existed at the time the leaf was created.  Since metadata indirectly upstream of a process may change without intermediary processes being rerun, the providence links are not simply the version of the direct input to a node at the time each metadata entity was created.  The upstream metadata change may have triggered a downstream rerun without intermediate processes being rerun.
+
+The knowledge if an update will result in a new or amended analysis is not known by the ingestion process.  It will be determined by each analysis examining the update.
+
+Figures 1-4 are some examples to illustrate the graph interpretation issues.  In each of these drawings, the graph node labels starting with B are biomaterials, those with P are processes, and with F are files.  The first number is the unique id, which is a UUID in practice.  The number after the dot is the version, which is represented as a timestamp.  Red squiggles indicate an outdated metadata instance.
+
+
+![Figure 1: MISSING](../images/0000-basic-graph.png)
+
+Figure 1: Basic graph of sequencing and analysis.  This is a basic experimental pipeline with no updates. The head graph is obtained by traversing the graph from the root to the leaves.  The provenance of a given leaf is achieved by reverse traversing the edges.
+
+
+![Figure 2: MISSING](../images/0000-updated-analysis-graph.png)
+
+Figure 2: Graph with an updated analysis.  In this case, an analysis is rerun for a reason other than an upstream change.  For example, this could be due to a bug be discovered in the analysis software.  The head graph is obtained by traversing the graph, ignoring the outdated nodes.  The provenance graph is obtained by reverse traversing the edges shown in red.
+
+![Figure 3: MISSING](../images/0000-new-analysis-graph.png)
+
+Figure 3: Graph with a new analysis:  In this case, a second, independent analysis is run.  This might be the case when a new different analysis algorithm is being used.  The head graph includes both analysis results.   The provenance graph is obtained by reverse traversing the edges shown in red.
```

Comment:
Should this:
> The provenance graph is obtained by reverse traversing the edges shown in red.

be:
The provenance graph is obtained by reverse traversing the edges shown in red **or black**.

Reply:
good point @Bento007 , I updated the cartoons to make the interpretation of the provenance graph clear.